packages/artifact_proxy/tool/generate_manifest.sh

```diff
@@ -48,6 +48,11 @@ artifact_overrides:
   - flutter_infra_release/flutter/\$engine/android-x64-release/symbols.zip
   - flutter_infra_release/flutter/\$engine/android-x64-release/windows-x64.zip
 
+  # Dart SDK
+  - flutter_infra_release/flutter/\$engine/dart-sdk-darwin-arm64.zip
+  - flutter_infra_release/flutter/\$engine/dart-sdk-linux-x64.zip
+  - flutter_infra_release/flutter/\$engine/dart-sdk-windows-x64.zip
```

Comment:
Do we need to include other archs? (eg `darwin-x64`)

Reply:
Added. We'll need to make sure we build this (I don't think we currently have the infra to do so)